build/3-apillon-api.md

```diff
@@ -1190,31 +1191,32 @@ API is for creating and managing NFTs. To prepare images and metadata you can us
-| description      | `string`  | NFT collection description.                                                                       |
-| maxSupply        | `number`  | Maximal number of NFTs ever in existence.                                                         |
-| bucketUuid       | `string`  | UUID of the bucket where metadata is stored.                                                      |
-| baseUri          | `string`  | Base URI for collection metadata (token id and file extension is appended to it).                 |
-| baseExtension    | `string`  | File extension that is auto appended after token id to form a full URL.                           |
-| isSoulbound      | `boolean` | Soul bound tokens are NFTs that are bounded to wallet and not transferable.                       |
-| isRevokable      | `boolean` | For revocable collection owner can destroy NFTs at any time.                                      |
-| royaltiesFees    | `number`  | Percentage (between 0 and 100) of each NFT sale sent to wallet specified under royalties address. |
-| royaltiesAddress | `string`  | Address where royalties are sent to.                                                              |
-| chain            | `number`  | Blockchain id on which you want to release your collection.                                       |
-| contractAddress  | `string`  | Smart address of contract for deployed collection.                                                |
-| transactionHash  | `string`  | Deployment transaction hash/id.                                                                   |
-| deployerAddress  | `string`  | Wallet address of deployer.                                                                       |
-| drop             | `boolean` | Determines if collection is mintable by public.                                                   |
-| dropStart        | `number`  | UNIX timestamp which determines public mint opening date and time.                                |
-| dropPrice        | `number`  | Price of NFT at mint stage.                                                                       |
-| dropReserve      | `number`  | Amount of NFTs reserved by owner.                                                                 |
+| Name             | Type        | Description                                                                                       |
+|------------------|-------------|---------------------------------------------------------------------------------------------------|
+| createTime       | `DateTime`  | Collection create time.                                                                           |
+| updateTime       | `DateTime`  | Collection last update time.                                                                      |
+| collectionType   | `number`    | Type of smart contract to use for collection (1 for generic, 2 for nestable).                     |
```

Comment:
Instead of writing 1 for generic, 2 for nestable directly link to the table explaining types.